umbraco-cloud/upgrades/migrate-from-umbraco-7-to-8.md

```diff
@@ -0,0 +1,171 @@
+---
+description: "This article will provide detailed steps on how to migrate a Umbraco 7 Cloud project to Umbraco 8."
+---
+
+# Migrate from Umbraco 7 to Umbraco 8 on Umbraco Cloud
+
+{% hint style="warning" %}
+Due to [a known issue](https://github.com/umbraco/Umbraco-CMS/issues/7914), it is **not possible to directly migrate your Umbraco 7 project to the latest Umbraco 8**.
+
+As a workaround, you can either
+
+* Migrate to **version 8.5** as a first step, and then, post-migration, carry out a normal Umbraco upgrade to the latest version of Umbraco 8, or
+* Install the following community Nuget Package: [ProWorks Umbraco 8 Migrations](https://www.nuget.org/packages/ProWorks.Umbraco8.Migrations) into your V8 project before running the migration (no configuration required). [Learn more about how to use this package on Prowork's blog](https://www.proworks.com/blog/archive/how-to-upgrade-umbraco-version-7-to-version-8).
+
+The package mentioned above patches the migration process. This means you can migrate directly from the latest Umbraco 7 to the latest Umbraco 8 version without encountering issues. The package is currently not tested on Umbraco Cloud.
+{% endhint %}
+
+Read the [general article about Content migration](../../umbraco-cms/fundamentals/setup/upgrading/version-specific/migrate-content-to-umbraco-8.md#what-are-the-limitations) to learn more about limitations and other things related to migrating your Umbraco site from 7 to 8.
+
+## Video tutorial
+
+<iframe width="800" height="450" title="Migrate an Umbraco Cloud project from 7 to 8: Introduction" src="https://www.youtube.com/embed/wNIDdgdAt8s?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+You can find the full playlist here: [Migrate an Umbraco Cloud project from 7 to 8](https://www.youtube.com/playlist?list=PLgX62vUaGZsGceCXveMkprlDV5a3K94db)
+
+## Prerequisites
+
+* A Umbraco 7 Cloud project running **the latest version of Umbraco 7**
+* Make sure Umbraco Forms data is not handled as content
+  * See [Umbraco Forms on Cloud](../deployment/umbraco-forms-on-cloud.md#how-forms-are-handled-on-umbraco-cloud) for more details on how to check the setting
+* A clean Cloud project running the latest version of Umbraco 8 with **at least 2 environments**
```

Comment:
Again this is related to the above comment, the top warning says that it is not possible to upgrade to the latest version of Umbraco 8, while the prerequisite is that the v8 project needs to be the latest v8 version, which seems contradictory 🤔

Reply:
Following the comment above - the first prerequisites would instead be "A Umbraco 7 project upgraded to Umbraco 8.5 locally".

Then we could add a link to the (on-prem) migration docs before this migration starts.

Reply:
Yeah i agree, it makes sense to change the prerequisite :)